src/web/js/trove/world.js

```diff
@@ -15,6 +15,8 @@
     values: {
       "reactor": ["forall", ["a"], ["arrow", [["tid", "a"], ["List", "WCOofA"]], "Any"]],
       "big-bang": ["forall", ["a"], ["arrow", [["tid", "a"], ["List", "WCOofA"]], ["tid", "a"]]],
+      "_patch_big-bang": "tany",
```

Comment:
Again, really don't like exposing `_pseudoprivate` names.  What are these for?

Reply:
Yeah, Patch's big-bang procedure is not quite the same as Pyret's, although it uses the same name. There are quite a few such procedures where Patch prefers a slightly different signature, as you've noticed.

I used _patch_big-bang as the name of the Pyret procedure that Patch's big-bang would map to. It is a bona-fide procedures and _could_ be used in Pyret (i.e., they won't cause errors if used with appropriate arguments, so it's not really private). I can update it and similar procedures to use your new-image-lib eventually so as not to clutter the Pyret namespace. The naming convention (they all start with _patch_).